packages/node_modules/@ciscospark/react-redux-spark/src/helpers.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable import/prefer-default-export */
+
+/**
+ * Gets the status booleans from different parts of the SDK instance
+ *
+ * @export
+ * @param {Object} sdkInstance
+ * @returns {Object} status
+ * @returns {boolean} status.authenticated
+ * @returns {boolean} status.authenticating
+ * @returns {boolean} status.registered
+ */
+export function getStatusFromInstance(sdkInstance) {
```

Comment:
I like the simplified shape of this interface over the current SDK's status object.

Reply:
Yea, it might be helpful to add these properties to the main webex core object in the future.